### PR TITLE
chore: bump kommander/opsportal ui to 6.84.1

### PR DESCRIPTION
**Pull request by natmegs**
**What type of PR is this?**
Chore

**What this PR does/ why we need it**:
Bump UI for kommander/opsportal

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by gracedo — APPROVED** [review]

File: stable/opsportal/requirements.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kommander-ui
-    version: "6.81.7"
+    version: "6.84.1"
```
Comment (alejandroEsc) [inline stable/opsportal/requirements.yaml:3]: this is reverting?

**Review by alejandroEsc — COMMENTED** [review]
one question, no blockerrs.

File: stable/opsportal/requirements.yaml (hunk 1 above)
Comment (natmegs) [reply stable/opsportal/requirements.yaml:3]: no, upgrading 81 -> 84? unless I messed this up, I've looked at too many version numbers today

**Review by samvantran — APPROVED** [review]

File: stable/opsportal/requirements.yaml (hunk 1 above)
Comment (gracedo) [reply stable/opsportal/requirements.yaml:3]: lgtm

**Pull request merged**